Good Coding Practices/src/js/scripts.js

```diff
@@ -1,41 +1,106 @@
 // piece object
-const piece = (function() {
-  let el = null;
-  const init = function(el) {
+const piece = (function () {
+  // let el = null;
+  const init = function (el) {
     this.el = el;
+    saveDefaultPositin.call(this);
+    setColorByTemperature.call(this);
   };
-  const moveDelta = function(dx, dy) {
+
+  const reset = function () {
+    const posLeft = localStorage.getItem("defaultPosLeft");
+    const posTop = localStorage.getItem("defaultPosTop");
+
+    this.el.style.left = posLeft + getPixelSuffix();
+    this.el.style.top = posTop + getPixelSuffix();
+  };
+
+  const setColorByTemperature = function (el) {

```

Comment:
Does the `piece` object need to know there's an API? 
Does it need to know what is the API's URL?
How is its data structured?